cpp/cppsdk/gsdk.cpp

```diff
@@ -155,6 +155,11 @@ namespace Microsoft
                     logFolder = "";
                 }
 
+                if (logFolder.back() != '/' && logFolder.back() != '\\')
```

Comment:
```suggestion
                if (!logFolder.EndsWith(System.IO.Path.DirectorySeparatorChar) && !logFolder.EndsWith(System.IO.Path.AltDirectorySeparatorChar))
```

Reply:
std::string doesn't offer a function called "EndsWith(...)"

Reply:
Ugh, you're right, it's 5.0+.
https://docs.microsoft.com/en-us/dotnet/api/system.string.endswith?view=net-5.0
Let's just switch the slashes to System.IO.Path.DirectorySeparatorChar then. Thanks & Sorry.